packages/tokens/src/figma-tokens/_docs.json

```diff
@@ -773,6 +763,54 @@
         "value": "Visually Hidden",
         "type": "text"
       }
+    },
+    "radio": {
+      "default": {
+        "title": {
+          "value": "Radio",
+          "type": "text"
+        },
+        "description": {
+          "value": "Radios allow the user to select a single option from a group. \nRadios are designed to be used with radio groups.",
+          "type": "text"
+        }
+      },
+      "examples": {
```

Comment:
In https://github.com/SickDesignSystem/synergy/pull/112/files#diff-35864d9aee16b937faed6a2d8d9b635446c65983c9c77c1bbd86b8dac5d2e16cR749, there is a mention of a label property for text fields and areas, which makes completely sense. However, I think that for the sake of completeness, radio buttons should also allow a label to be used. Either it is not documented or missing from the component at all.

@mariohamann : What do you think about this?

Reply:
It's in the group.

https://shoelace.style/components/radio-group

Reply:
+ radios theirself have a label, too. It's just the default slot there.

Reply:
Yes, Shoelace does it providing the `name` attribute in the group: https://shoelace.style/components/radio-group

So it would be:

```
<form>
  <syn-radio-group name="test">
    <syn-radio value="item1">Item 1</syn-radio>
    <syn-radio value="item2" checked>Item 2</syn-radio>
    <syn-radio value="item2">Item 3</syn-radio>
  </syn-radio-group>
  <syn-button type="submit">Submit form</syn-button>
</form>
```

Reply:
Ok, that not quite as standard compliant as I was hoping for, but I see where this comes from and it would be more of a pain to eventually adopt it to make it work like it should.